.github/workflows/docs.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -e .[docs]
+          pip install sphinx-typo3-theme==4.4.3
```

Comment:
This dependency should be defined in setup.py, as well as `docs/requirements.txt`, and removed from here.